client/src/components/PreviousFeedback/styles.css

```diff
@@ -17,4 +17,15 @@ justify-content: flex-end;
 .previous-feedback-title-and-counter{
```

Comment:
Is this supposed to be called container instead of counter?

Reply:
Will make the change. You are right